src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -585,21 +577,30 @@ Where:
 - `elasticsearch.clusterName` is the cluster name to join, `elasticsearch` being the default cluster name.
 
 #### Advanced REST Client configuration
-If you have installed [Elasticsearch X-Pack](https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html) you have the possibility to secure communication between Nuxeo and Elasticsearch using the Rest Client (supported since Nuxeo 9.10-HF01).
 
-For Elasticsearch please follow this guide to [Securing Elasticsearch and Kibana](https://www.elastic.co/guide/en/x-pack/5.6/xpack-security.html).
+##### Elasticsearch 7
+
+Elasticsearch requires [the X-Pack extension](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/setup-xpack.html) to enable secured communication between Nuxeo and Elasticsearch using the Rest Client (supported since Nuxeo 9.10-HF01).
+
+Please follow this guide to [Securing Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/security-settings.html).
 
-##### Basic Authentication
-If you have chosen to configure [Basic User Authentication](https://www.elastic.co/guide/en/x-pack/5.6/security-getting-started.html) then you can setup Nuxeo using `nuxeo.conf` with the follow properties:
+##### Elasticsearch 8
+
+Security is enabled by default. [Further security configuration can be found here](https://www.elastic.co/guide/en/elasticsearch/reference/8.7/security-settings.html)
+
+##### Basic Security and Authentification
+If you have chosen to configure [Basic security](https://www.elastic.co/guide/en/elasticsearch/reference/8.7/security-basic-setup.html) then you can setup Nuxeo using `nuxeo.conf` with the follow properties:
 
 ```
 elasticsearch.restClient.username=your_username
 elasticsearch.restClient.password=your_password
 ```
 {{#> callout type='tip' }}
-For X-Pack, please follow the Elasticsearch documentation for configuring a user and role, an example could be:
+Please follow the [Elasticsearch REST Security APIs documentation](https://www.elastic.co/guide/en/elasticsearch/reference/8.7/security-api.html) for configuring a [user](https://www.elastic.co/guide/en/elasticsearch/reference/8.7/security-api-put-user.html) and [role](https://www.elastic.co/guide/en/elasticsearch/reference/8.7/security-api-put-role.html). 
```

Comment:
this should be given in the 7.x and 8.x section not in the common section about basic auth which should be just the nuxeo conf which identical for both versions.